rust/connlib/snownet/src/backoff.rs

```diff
@@ -1,31 +1,51 @@
 use std::time::{Duration, Instant};
 
-pub type ExponentialBackoff = backoff::exponential::ExponentialBackoff<ManualClock>;
+const MULTIPLIER: f32 = 1.5;
+const MAX_ELAPSED_TIME: Duration = Duration::from_secs(8);
```

Comment:
Was this change from 10s to 8s intentional?

Reply:
The new implementation aborts the backoff when it would exceed this `MAX_ELAPSED_TIME`, i.e. The previous one would allow for one tick after that.

You can see in the test that compares against `steps` that we still have the same number of backoff steps.

Reply:
👍